Sources/SwiftFormatConfiguration/Configuration.swift

```diff
@@ -88,6 +88,15 @@ public class Configuration: Codable {
   /// each argument, forcing the entire argument list to be laid out vertically.
   public var lineBreakBeforeEachArgument = false
 
+  /// Determines if function-like declaration outputs should be prioritized to be together with the
+  /// function signature right (closing) parenthesis.
+  ///
+  /// If false (the default), function output (i.e. throws, return type) is not prioritized to be
+  /// together with the signature's right parenthesis, and when the line length would be exceeded,
+  /// a line break will be fired after the function signature first, indenting the declaration output
+  /// one additional level. If true, A line break will be fired further up in the function's
+  /// declaration (e.g. generic parameters, parameters) before breaking on the function's output.
+  public var prioritizeKeepingFunctionOutputTogether = false
   /// Determines the indentation behavior for `#if`, `#elseif`, and `#else`.
```

Comment:
Nit: Add a blank line after this property.

Reply:
Oops. That's what you get by splitting commits 😅. Fixed

Reply:
Hadn't pushed any changes yet. Sorry for the miscommunication. Should be by now 😄